src/common/datetime/format_duration.ts

```diff
@@ -1,16 +1,23 @@
 import { HaDurationData } from "../../components/ha-duration-input";
+import { HomeAssistant } from "../../types";
 
 const leftPad = (num: number) => (num < 10 ? `0${num}` : num);
 
-export const formatDuration = (duration: HaDurationData) => {
+export const formatDuration = (
+  hass: HomeAssistant,
+  duration: HaDurationData
+) => {
   const d = duration.days || 0;
   const h = duration.hours || 0;
   const m = duration.minutes || 0;
   const s = duration.seconds || 0;
   const ms = duration.milliseconds || 0;
 
   if (d > 0) {
-    return `${d} day${d === 1 ? "" : "s"} ${h}:${leftPad(m)}:${leftPad(s)}`;
+    return `
+      ${hass.localize("ui.duration.day", { count: d })}
+      ${h}:${leftPad(m)}:${leftPad(s)}
+    `;
```

Comment:
```suggestion
    return hass.localize("ui.duration.day", { count: d }) + `${h}:${leftPad(m)}:${leftPad(s)}`;
```

Reply:
We generally prefer template strings above concatenation, it is easier to read and causes less mistakes (like you forgot the space between day and hours there)

Reply:
Yeah but the original has a load of whitespace, what if it gets passed to somewhere that doesn't do whitespace collapsing?